.github/workflows/CI.yml

```diff
@@ -76,6 +76,8 @@ jobs:
         shell: bash
       - name: Run tests
         run: npm test
+      - name: Run examples
+        run: npm run examples
```

Comment:
IIUC, examples contain no assertions, so they don't test anything using assertions. What do we get by running examples in CI? We only (but it's still some benefit) know that they run without throwing exceptions, correct?

Reply:
That's correct.